examples/resources/awscc_cloudformation_stack_set/service_managed_stackset.tf

```diff
@@ -0,0 +1,28 @@
+resource "awscc_cloudformation_stack_set" "stackset" {
+  stack_set_name = "network-stackset1"
```

Comment:
is there reason this has to be named as `network-stackset1` ?

Reply:
Not really. I was working with 2 examples with self managed and service managed and had to differentiate while testing. Will change it to `network-stackset`